crates/hc_run_local_services/Cargo.toml

```diff
@@ -10,7 +10,7 @@ categories = [
   "development-tools::build-utils",
   "filesystem",
 ]
-edition = "2021"
+edition = "2024"
```

Comment:
Are we not reverting this in holochain for the time being?

Reply:
Yes, this will need reverting when K2 is next published. Hoping to get the bootstrap client in over there first